org.jellyfin.webos/css/main.css

```diff
@@ -16,8 +21,8 @@ body {
     width: 100%;
     font-size: 36px;
     font-family: -apple-system, "Helvetica", system-ui, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", "Open Sans", sans-serif;
```

Comment:
Not really necessary to do here, but this line seems a bit weird since WebOS doesn't have most of these fonts.
It could probably be removed?

Or at least simplified. Not blocking though.

Reply:
Yeah I imagine it probably could be simplified, but I have no idea what kind of font stacks are shipped on webos systems.